client/apps/PageContent.tsx

```diff
@@ -3,6 +3,7 @@ import classNames from 'classnames';
 
 import {WorkqueueContainer, ModalsContainer} from '../components';
 import {PopupEditorPortal} from '../components/Main/ItemEditorModal';
+import {gettext} from '../utils';
```

Comment:
This was the older way of using `gettext` in Planning.
The newer way is to use the `superdeskApi` instead (which allows Superdesk to control how translations are implemented/provided).

```ts
import {superdeskApi} from '../superdeskApi';

class Test extends React.PureComponent {
    render() {
        const {gettext} = superdeskApi.localization;

        return (
            <span>{gettext('Doing translations the newer way')}</span>
        );
    }
}
```

There's no need to change imports for **all** `gettext`, but if we're adding a new import of `gettext` it would be beneficial to use the `superdeskApi` version.